_layouts/report.md

```diff
@@ -82,6 +82,14 @@
                             <div class="center"><h5 class="light no-margin">Patches Tarball</h5></div>
                             <br>
                             <div class="center"><a href="{{ page.tarball }}">Download</a></div>
+                            <br>
+                            <div class="center sha256sum_hash">
+                                <strong>SHA-256</strong>:
+                                <p id="sha256sum_hash_value">
+                                    {{page.sha256sum}}
+                                </p>
+                                <i class="fa fa-clipboard" aria-hidden="true"></i>
```

Comment:
Can you add hover effect here - at least the cursor should become a pointer when hovering.

Reply:
Will commit soon with hover effect on button :-)

Reply:
![screenshot 40](https://user-images.githubusercontent.com/35761292/48262318-0fe14580-e448-11e8-9647-f7875a12ac65.png)
On hovering over icon, the cursor changes to a pointer and a hint text is being displayed. Looks Good to me

Reply:
Looks good, can you push your changes 👍

Reply:
Pushed my changes 😃